server/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
maybe version 3.5? like in rest cmakelists?

Reply:
ok I will change it